src/main/java/uk/gov/companieshouse/web/accounts/model/smallfull/BalanceSheet.java

```diff
@@ -21,4 +21,6 @@
 
     @Valid
     private CurrentAssets currentAssets;
+
+    private OtherLiabilitiesOrAssets otherLiabilitiesOrAssets;
```

Comment:
Should this field be annotated with `@Valid`?

Reply:
The `@Valid` annotation would be redundant here as we don't associate any other validation annotations with the field or nested object fields. For consistency I've removed all of the unnecessary annotations.